pkgs/development/python-modules/ismartgate/default.nix

```diff
@@ -1,66 +1,58 @@
 { lib
-, asynctest
 , buildPythonPackage
-, click
-, defusedxml
-, dicttoxml
 , fetchFromGitHub
-, httpx
+, pythonOlder
+  # build inputs
```

Comment:
I'm not a fan of comments and grouping here because most approaches are not widely used or established among the maintainers. Everything which makes the sorting task a manual one should be avoided from my point of view. At least as long that the tool-based solution to do it is missing.